frontend/src/static/js/v_ramp.js

```diff
@@ -57,6 +36,42 @@ window.vRamp = {
       const newSize = 100 * (slice.insertions / this.avgsize);
       return Math.max(newSize * this.rampSize, 0.5);
     },
+    getTitle(slice, commit) {
+      let title = '';
+      if (this.tframe === 'commit') {
+        return `[${slice.date}] ${commit.messageTitle}: ${commit.insertions} lines`;
+      }
+
+      if (this.zoomfromramp) {
+        title = this.tframe === 'day'
+            ? `[${slice.date}] `
+            : `Week of ${slice.date} `;
+        title += `${commit.messageTitle}: ${commit.insertions} lines`;
+        return title;
+      }
+
+      title = this.tframe === 'day'
+            ? `[${slice.date}] Daily `
+            : `[${slice.date} till ${slice.endDate}] Weekly `;
+      title += `contribution: ${slice.insertions} lines`;
+      return title;
+    },
+    openTabZoom(user, slice, evt) {
+      if (window.isMacintosh ? evt.metaKey : evt.ctrlKey) {
```

Comment:
can put a comment that states the skipping of opening zoom tab when cmd/ctrl click